src/main/java/org/folio/ed/service/StagingDirectorIntegrationService.java

```diff
@@ -31,6 +32,7 @@
 
 @Service
 @RequiredArgsConstructor
+@Slf4j
 public class StagingDirectorIntegrationService {
   private static final Logger LOGGER = LoggerFactory.getLogger(StagingDirectorIntegrationService.class);
 
```

Comment:
no need to use Loggerfactory , use @Log4j2 and use logger.error / info directly

Reply:
changed to @log4j2